libs/components/src/tooltip/vl-tooltip.component.ts

```diff
@@ -1,10 +1,10 @@
-import { BaseElementOfType, webComponent } from '@domg-wc/common-utilities';
+import { type VL, BaseElementOfType, webComponent } from '@domg-wc/common-utilities'; // FIXME: Circular dependency ERROR <eslint@nx/enforce-module-boundaries>
```

Comment:
Is deze FIXME de bedoeling?

Reply:
gedaan. ik heb momenteel verwijderd.

maar soms krijg ik een eslint/@nx error -  van circular dependencies error -  op mijn IDE, andere momenten niet.